Formula/b/bob-nvim.rb

```diff
@@ -0,0 +1,32 @@
+class BobNvim < Formula
```

Comment:
why not just `bob`?

Reply:
note, other package managers are using `bob`, https://repology.org/project/bob/versions

Reply:
I agree, this should be `bob`.

Reply:
Cool, renamed. Wasn't sure if `bob` alone was unique enough.

Reply:
I think `bob` is pretty unique (so is [alice](https://repology.org/project/alice/versions))